kettle-plugins/hbase/src/main/java/org/pentaho/big/data/kettle/plugins/hbase/output/HBaseOutputDialog.java

```diff
@@ -854,15 +853,20 @@ private void getData() {
       defaultConf = transMeta.environmentSubstitute( m_defaultConfigText.getText() );
     }
 
-    NamedCluster nc = namedClusterWidget.getSelectedNamedCluster();
-    if ( nc != null ) {
-      zookeeperHosts = transMeta.environmentSubstitute( nc.getZooKeeperHost() );
-    }
+    //    NamedCluster nc = namedClusterWidget.getSelectedNamedCluster();
```

Comment:
@vamshirajidi  please remove this commented code if it is not needed

Reply:
Instead of commenting the code, I have added a check to find whether this flow is for Knox or not.